ChangeLog

```diff
@@ -1,3 +1,46 @@
+# 2023-12-22 Version 3.1.0
+
+A new 3.1.0 release only 10 days after 3.0.0.
```

Comment:
I think we should drop the `only 10 days after` wording here. It might express, that we would work sloppy, which is not the case here (but that wording could implicate that here, since this is the first "minor" stable update after the first major release, in a short time, while that short time is emphasized here).

Reply:
good point. updated.
also added the `(optional)` for the new build dependencies more prominent